### PR TITLE
support --modelPath (-M) and --modelSpec (-m)

### PR DESCRIPTION
**Pull request by nlhepler**
r? @dalexander @armintoepfer

### CODE REVIEW

File: src/main/ccs.cpp [hunk 1]
```diff
@@ -542,7 +570,7 @@ int main(int argc, char** argv)
         const auto outputPrefix = outputFile.substr(0, outputFile.size() - 4);
         // Save dataset
         std::ofstream ccsOut(outputPrefix + ".consensusreadset.xml");
-        ccsSet.SaveToStream(ccsOut); 
+        ccsSet.SaveToStream(ccsOut);
```
Comment (dalexander) [inline src/main/ccs.cpp]: :-)

**Comment by dalexander** [comment]
The diffs look good... does it work?

File: src/main/ccs.cpp [hunk 2]
```diff
@@ -466,6 +473,19 @@ int main(int argc, char** argv)
         Logging::InstallSignalHandlers();
     }
 
+    // load models from file or directory
+    //
+    //
+    {
+        string modelPath(options.get(OptionNames::ModelPath));
+        if (!modelPath.empty()) {
+            if (!PacBio::Consensus::LoadModels(modelPath)) {
```
Comment (dalexander) [inline src/main/ccs.cpp]: I recommend logging here, this is important for knowing what's happening

File: src/main/ccs.cpp [hunk 3]
```diff
@@ -478,23 +498,31 @@ int main(int argc, char** argv)
...
-        try {
-            used = ds.SequencingChemistries();
-        } catch (InvalidSequencingChemistryException& e) {
-            PBLOG_FATAL << e.what();
-            exit(-1);
-        }
-        vector<string> unavail;
+        string modelSpec(options.get(OptionNames::ModelSpec));
+        if (!modelSpec.empty()) {
+            if (!(PacBio::Consensus::OverrideModel(modelSpec) && used.insert(modelSpec).second)) {
```
Comment (dalexander) [inline src/main/ccs.cpp]: Log here?

**Comment by nlhepler** [comment]
We can test this when S/P1-C1/beta is working

File: src/main/ccs.cpp (hunk 1 above)
Comment (nlhepler) [inline src/main/ccs.cpp]: Noooooo

**Comment by dalexander** [comment]
Awesome

**Pull request merged**